velox/common/config/tests/ConfigTest.cpp

```diff
@@ -157,6 +157,10 @@ TEST_F(ConfigTest, immutableConfig) {
   ASSERT_TRUE(config->get<int32_t>(TestConfig::kInt32Entry.key).has_value());
   ASSERT_EQ(config->get<int32_t>(TestConfig::kInt32Entry.key).value(), -3200);
   ASSERT_FALSE(config->get<int32_t>("wrong_int32_key").has_value());
+  ASSERT_EQ(config->get<int32_t>("wrong_int32_key", 100), 100);
```

Comment:
Could you clarify why this test is necessary in the current PR? I might have overlooked something.